include/boost/algorithm/is_clamped.hpp

```diff
@@ -0,0 +1,73 @@
+/* 
+   Copyright (c) Marshall Clow 2008-2012.
```

Comment:
Shouldn't this be your name since you authored the file?

Reply:
It is point 5. in my description, I really have no idea what to do here, I dont mind "gifting precious copyright" to Marshall for 3 lines of implementation, I just want it to be freely usable by people.

Reply:
@mclow  I think beside copyright and documentation this is ready for review. Could you provide some guidance wrt what to put in copyright, and what to do about documentation? Is is autogenerated from source comments or I need to edit some HTML file?

Reply:
You should absolutely put your name in as the copyright holder.  You wrote it, you should take credit.

The boost license is sufficient to make it "freely usable by people". You should read the license to make sure that you're OK with it.

Reply:
As for documentation, some will be generated by the doxygen comments that you have provided.  If you want more, then you'll have to write it.

Reply:
@mclow I put us both in Copyright, authors since this is mostly based on your work on clamp. Is this ok?
As for docs: I am aiming for same level of docs as clamp has.  So I would like to do something like this for is_clamped: https://www.boost.org/doc/libs/1_77_0/libs/algorithm/doc/html/algorithm/Misc.html#the_boost_algorithm_library.Misc.misc_inner_algorithms.clamp so  I will do similar as what clamp-hpp.qbk does.

 Do you know how I can preview HTML docs on my machine to see how it looks after I am done, I found quick book [docs](https://github.com/boostorg/quickbook/blob/develop/doc/install.qbk), but does not seem trivial, I was hoping I can issue some b2 command and it will output the HTML docs into some folder.

Reply:
You build the docs by `cd docs; ../../b2`.  But you need quickbook, etc installed.

Reply:
Hi @mclow  I have hit some problems with this. 
Building docs seems to fail.
[boost doc error.txt](https://github.com/boostorg/algorithm/files/7190208/boost.doc.error.txt)

And output of build seems to contain no html files. 
![image](https://user-images.githubusercontent.com/47703951/133894703-0b392d1f-6b67-4117-9a81-124ffc05efae.png)

maybe problem is that here option for HTML seems to be turned off?
![image](https://user-images.githubusercontent.com/47703951/133894768-c1c09ab7-7d2d-44d6-bae0-758145a72bf6.png)

Reply:
@mclow  could we merge this(if code beside docs is fine) without quickbook docs?

It is relatively simple algorithm.